template.js

```diff
@@ -71,9 +72,22 @@ return sendRequest(data.url, requestOptions, postBody).then(mapResponse);
 
 function sendRequest(url, requestOptions, postBody) {
   let cacheKey = sha256Sync(version + url + JSON.stringify(requestOptions) + postBody + data.jsonParseKeyName);
-
+  let cacheTimeKey = cacheKey + '_timestamp';
+  let timeNow = getTimestampMillis();
+  
   if (data.storeResponse) {
-    const cachedBody = templateDataStorage.getItemCopy(cacheKey);
+    let cachedBody = templateDataStorage.getItemCopy(cacheKey);
+    let cachedBodyTimestamp = templateDataStorage.getItemCopy(cacheTimeKey);
+    if (data.expirationTime) {
+      let expiratoinTimeSeconds = data.expirationTime * 360000; // convert to miliseconds
```

Comment:
makeInteger(data.expirationTime)

Reply:
Will add but in template its positive integer